### PR TITLE
Feature/add plant

### PR DESCRIPTION
**Pull request by KonradMierzejewski**
I need a little help on styling, I don't know how to style some elements (you can see it on AddPlant -> select and button components). If I missed some functionality, let me know.

### CODE REVIEW

File: src/containers/AddPlant/AddPlant.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+          <Text text="Help us grow!" fontsize="2em" />
+          <Text text="If you did not find your plant in our base, you can add it below." fontsize="2em" />
+        </div>
+        <Modal closeModal={closeModal} show={show} />
+        <Input className={styles.input} text="Name" />
+        <Input className={styles.input} text="Latin Name" />
+        <Select className={styles.select} title="Species" />
+      </div>
+      <div className={styles.detailedInfo}>
+        <Select className={styles.select} title="Min temperature" values={[10, 15, 20]} />
```
Comment (adax10) [inline src/containers/AddPlant/AddPlant.js]: Why do you enter class names for imported components? It doesn't make sense, if you want to style them appeal through children.

**Review by adax10 — CHANGES_REQUESTED** [review]
And logic is missing.

**Comment by kami3la** [comment]
- when you import, you do not have to double-write something in the paths because we have index files to make importing easier
- instead of importing `styles`, import specific styles
- change the name of the show style to another so that it does not repeat and in the file with styles also change this name and it should work
image

**Comment by jedrzejratajczak** [comment]
You don't have to import specific styles if it is problematic, for larger components there can be some naming issues with other variables

**Pull request closed without merging**